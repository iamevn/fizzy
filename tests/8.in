# with range omitted
4,17:no need for
11:a range
