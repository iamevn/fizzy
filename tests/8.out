1
2
3
no need for
5
6
7
no need for
9
10
a range
no need for
13
14
15
no need for
no need for
18
19
no need for
21
a range
23
no need for
25
26
27
no need for
29
30
31
no need for
a range
no need for
35
no need for
37
38
39
no need for
41
42
43
no need fora range
45
46
47
no need for
49
50
no need for
no need for
53
54
a range
no need for
57
58
59
no need for
61
62
63
no need for
65
a range
67
no need for
69
70
71
no need for
73
74
75
no need for
a range
78
79
no need for
81
82
83
no need for
no need for
86
87
no need fora range
89
90
91
no need for
93
94
95
no need for
97
98
a range
no need for
